### PR TITLE
Add monitoring module

### PR DESCRIPTION
**Pull request by ajaen4**
- Add monitoring module that creates a lambda that queries the data tables every minute to send metrics to CW
- Modify make file so locust is executed with correct tag

### CODE REVIEW

File: infra/vars.tf [hunk 1]
```diff
@@ -3,6 +3,10 @@ variable "aws_region" {
   default = "eu-west-1"
 }
 
+variable "aws_account_id" {
+  type    = string
+}
```
Comment (adrijh) [inline infra/vars.tf]: We do account id as a variable or we use aws_caller_identity? https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/caller_identity

File: infra/monitoring.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module "monitoring" {
+
+  source = "./monitoring"
+
+  aws_region_name = var.aws_region
+  aws_account_id = var.aws_account_id
+  image_tag = "0.0.1"
+  database_name = "hudi"
+  table_name = "ticker_hudi_mor_ro"
+}
```
Comment (adrijh) [inline infra/monitoring.tf:10]: Maybe add this to main.tf rather than a single file

Comment (ajaen4) [reply infra/monitoring.tf:10]: I prefer having it separately, right now it is now one resource for @alfonjerezi to have an example but there will be as many as tables you want to query for metrics.

**Review by adrijh — APPROVED** [review]

File: infra/monitoring.tf (hunk 2 above)
Comment (adrijh) [reply infra/monitoring.tf:10]: Ok that sounds good then

**Pull request merged**